Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -100,13 +106,51 @@ public BlackboardArtifactNode(BlackboardArtifact artifact) {
         this.setIconBaseWithExtension(ExtractedContent.getIconFilePath(artifact.getArtifactTypeID())); //NON-NLS
     }
 
+    @Override
+    @NbBundle.Messages({"BlackboardArtifactNode.getAction.errorTitle=Error getting actions",
+        "BlackboardArtifactNode.getAction.errorMessage=There was a problem getting actions for the selected result."})
```

Comment:
If you want to do a notification here, then it should be more informative, i.e., the View in Timeline action will not be available.